src/extension.ts

```diff
@@ -138,6 +147,58 @@ export async function activate(context: vscode.ExtensionContext) {
   console.log('Clang Language Server is now active!');
   fileStatus.activate(client, context);
   switchSourceHeader.activate(client, context);
+  async function handleConfigFilesChanged(uri: vscode.Uri) {
```

Comment:
could you move this part of new code to a dedicated file `config-changed.ts`? You can find the similar patterns in `switch-source-header.ts`, `file-status.ts`.